.pre-commit-config.yaml

```diff
@@ -68,18 +68,18 @@ repos:
         files: '^ote_cli/.*\.py'
         additional_dependencies: [types-PyYAML]
       - id: mypy
-        alias: mypy_external
-        name: "mypy (external)"
-        files: '^(external/anomaly|external/model-preparation-algorithm)/.*\.py'
+        alias: mypy_anomaly
+        name: "mypy (anomaly)"
+        files: '^external/anomaly/.*\.py'
```

Comment:
Any reason why only anomalies apply?

Reply:
As stated in the description, the MPA task currently has so many mypy & pilint issues. It is appropriate to solve this problem in the refactoring phase, and it is currently temporarily disabled in MPA to avoid affecting OTE CI results.

Reply:
Thanks for the explanation. Looks good for me.